### PR TITLE
[bitnami/solr] Adds support for TLS

### PR DESCRIPTION
**Pull request by rafariossaa**
**Description of the change**

Adds support for SSL in solr.
Needs a new docker solr image that supports SSL to be released.

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Review by miguelaeh — CHANGES_REQUESTED** [review]
Thank you very much for the PR!
Please take a look at the comments.
Also, please add documentation to the README about how to use SSL

**Comment by rafariossaa** [comment]
I moved back the liveness/readiness check to use internal httpGet.

File: bitnami/solr/README.md [hunk 1]
```diff
@@ -157,6 +157,15 @@ The following tables lists the configurable parameters of the solr chart and the
...
 | `resources`                            | Solr CPU/Memory resource requests/limits                     | `{Memory: "256Mi", CPU: "100m"}`   |
 
+### Solr SSL parameters
+
+| Parameter                              | Description                                                  | Default                            |
+| -------------------------------------- | ------------------------------------------------------------ | ---------------------------------- |
+| `tls.enabled`                          | Enable the TLS/SSL configuration                             | `false`                            |
+| `tls.certificatesSecretName`           | Set the name of the secret that contains the certificates. It should contains two keys called "keystore.p12" and "truststore.12"                                                  | `nil`                             |
+| `tls.passwordsSecretName`              | Set the name of the secret that contains the passwords for the certificate files. It should contains two keys called "tls-keystore-password" and "tls-truststore-password".       | `nil`                             |
+
```
Comment (miguelaeh) [inline bitnami/solr/README.md:167]:
```suggestion
```

**Review by miguelaeh — APPROVED** [review]
LGTM!
Delete the empy line first

**Comment by rafariossaa** [comment]
Do not merge until the new container image is released.

**Pull request merged**